core/gui/src/app/workspace/component/workspace.component.ts

```diff
@@ -119,16 +120,28 @@ export class WorkspaceComponent implements AfterViewInit, OnInit, OnDestroy {
   }
 
   ngOnDestroy() {
+    this.saveLatestWorkflow();
     this.workflowActionService.setWorkflowMetadata(undefined);
     this.workflowActionService.destroySharedModel();
     this.workflowWebsocketService.closeWebsocket();
   }
 
   @HostListener("window:beforeunload")
```

Comment:
@aglinxinyuan can we move this annotation on to `ngOnDestroy`?

Reply:
Yes.

Reply:
Merged